src/api/teams.rs

```diff
@@ -0,0 +1,62 @@
+//! The Teams API
+
+mod children;
+mod create;
+mod edit;
+mod list;
+mod team_repos;
```

Comment:
The structs inside this need to publicly re-exported to viewable in documentation. The modules should still be private though.